### DISCUSSION TITLE
support jsonb in sqlite

### DISCUSSION THREAD

**Discussion by graf0** [jeremyevans/sequel#2133] [General]
like subject states - support newest jsonb in sqlite_json_op functions on BLOB fields in sqlite >= 3.45.0

**Comment by jeremyevans** [comment]
This makes sense.  I'll see if I can work on this before the next release.

**Comment by graf0** [comment]
Operators -> and ->> just work. We need few more functions in jsonb variant in sqlite_json_op - that's it.

**Comment by jeremyevans** [comment]
I implemented the changes in f91225323e29f161c6ba5f08dd9398022bc84fb3
